e107_core/templates/usersettings_template.php

```diff
@@ -77,7 +77,7 @@
 $sc_style['LOGINNAME']['post'] 			= "</td></tr>\n";	
 	
 
-$sc_style['SIGNATURE']['pre']			= "<tr><td style='width:40%;vertical-align:top' class='forumheader3'>".LAN_USER_71.req(varset($pref['signup_option_signature']))."</td>
+$sc_style['SIGNATURE']['pre']			= "<tr><td style='width:40%;vertical-align:top' class='forumheader3'>".LAN_USER_71.req(varset(e107::getPref('signup_option_signature')))."</td>
 											<td style='width:60%' class='forumheader2'>";
```

Comment:
Not sure we need varset() if using getPref() ?

Reply:
The more I think about it, varset() can't even work, because the argument isn't a variable. I'll check it later.